src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -81,7 +95,8 @@ function CardSection() {
         BillingBanner = <TrialStartedBillingBanner />;
     } else if (SubscriptionUtils.hasUserFreeTrialEnded()) {
         BillingBanner = <TrialEndedBillingBanner />;
-    } else if (billingStatus) {
+    }
+    if (billingStatus) {
```

Comment:
Is this `else if` --> `if` intentional?

Are we saying along with Trial/Trial ended banner we would show the authenticate payment button?

Reply:
I think I got the change now.